### DISCUSSION TITLE
why skywalking choosing graphQL not other?

### DISCUSSION THREAD

**Discussion by QQQJoker** [apache/skywalking#11054] [Q&A] [answered]
Why skywalking chooses graphQL as the query protocol, and it is not restful or other

**Comment by kezhenxu94** [comment, marked as answer]
> Why skywalking chooses graphQL as the query protocol, and it is not restful or other 

There is no point to ask why technology A instead of technology B in open source projects, you can build a query plugin with your favorite technology, but don’t ask 🤫
